templates/vue/src/components/Lightbox/media/TapestryMedia.vue

```diff
@@ -143,10 +146,14 @@ export default {
       this.$emit("load", args)
     },
     updateProgress({ amountViewed }) {
-      this.updateNodeProgress({ id: this.nodeId, progress: amountViewed })
+      if (!Helpers.nodeAndUserAreDyad(this.node)) {
+        this.updateNodeProgress({ id: this.nodeId, progress: amountViewed })
+      }
     },
     complete(nodeId) {
-      this.$emit("complete", nodeId)
+      if (!Helpers.nodeAndUserAreDyad(this.node)) {
```

Comment:
Same as above, I would just move this check to the `completeNode` action itself.

Reply:
Yep, moved the checks into completeNode